src/components/clusters/clusterConfig.js

```diff
@@ -426,6 +422,18 @@ var snapTask = {
         "/intel/procfs/meminfo/mem_total_perc": {},
         "/intel/procfs/meminfo/mem_used": {},
         "/intel/procfs/meminfo/mem_used_perc": {},
+        "/intel/procfs/filesystem/*/inodes_free": {},
```

Comment:
the filesystem metrics need snap-plugin-collector-df, but i dont see it included in the image.  https://github.com/grafana/snap_k8s/pull/4/files

Reply:
Sorry about that I had pushed snap_k8s with the df plugin but I didn't push the commit yet